packages/grid/_modules_/grid/models/params/gridCellParams.ts

```diff
@@ -52,13 +55,16 @@ export interface GridCellParams {
    * @param id
    * @param field
    */
-  getValue: (id: GridRowId, field: string) => GridCellValue;
+  getValue: (id: GridRowId, field: string) => T;
```

Comment:
The same situation from the `formattedValue`. Calling `getValue` allows to get the value from other fields, whose type may differ from `T`. We could add another generic, but I think there're enough. Reverting this line is fine.

```suggestion
  getValue: (id: GridRowId, field: string) => GridCellValue;
```